sdk/provisioning/Azure.Provisioning.CloudMachine/tests/CloudMachineTests.cs

```diff
@@ -0,0 +1,42 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+#nullable enable
+
+using System;
+using Azure.Provisioning.CloudMachine;
+using NUnit.Framework;
+
+namespace Azure.CloudMachine.Tests;
+
+public class CloudMachineTests
+{
+    [Theory]
+    [TestCase([new string[] { "--init" }])]
+    [TestCase([new string[] { "" }])]
+    public void Configure(string[] args)
+    {
+        if (CloudMachineInfrastructure.Configure(args, (cmi) => {
+        })) return;
+
+        CloudMachineClient cm = new();
+        Console.WriteLine(cm.Id);
+    }
+
+    [Theory]
+    [TestCase([new string[] { "--init" }])]
+    [TestCase([new string[] { "" }])]
+    public void Storage(string[] args)
```

Comment:
Should we Ignore these live tests for now? Otherwise, they'll need to be setup as proper Recorded tests.

Reply:
Consider ignoring the entire class so it's less work to flip them all back on when you want to test locally.